hyp3_gamma/__main__.py

```diff
@@ -213,6 +221,13 @@ def water_map():
             f'--hand-threshold {args.hand_threshold} --hand-fraction {args.hand_fraction} '
             f'--membership-threshold {args.membership_threshold}', uselogging=True)
 
+    if args.include_flood_depth:
+        execute(f'conda run -n  asf-tools flood_map {product_name}/{product_name}_FM.tif '
```

Comment:
This command needs to be updated since `flood_map` is no longer a separate command.

Reply:
Nope, I'm dumb.